pages/osx/osascript.md

```diff
@@ -0,0 +1,27 @@
+# osascript
+
+> Run AppleScript or JavaScript for Automation (JXA) from the command line.
```

Comment:
"JavaScript for Automation"/"JXA" are the _official_ [silly] names given by Apple for their [shoddy] JavaScript implementation… again, not decisions made on my part! Whereas "AppleScript" is just called "AppleScript".